### PR TITLE
Homework 6 completed

### PR DESCRIPTION
**Pull request by obitkin**
### New profile:

- JdiSite

Example: `mvn test -P JdiSite` 

### Result

_Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 91.565 s_

### Browse and Driver requirements

- Chrome
- WebDriver driver is automatically downloaded by driver manager

### CODE REVIEW

File: src/main/java/hw6/site/elements/LoginForm.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+@Getter
+public class LoginForm extends Form<UserData> {
+
+    @Css("#name")
+    @Mandatory TextField login;
+
+    @Css("#password")
+    @Mandatory TextField password;
+
+    @Css("[type=submit]") Button loginButton;
```
Comment (troe88) [inline src/main/java/hw6/site/elements/LoginForm.java:20]: Code convention rule breaker!

File: src/main/java/hw6/site/pages/MetalsColorsPage.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import hw6.site.elements.Header;
+import hw6.site.elements.LinesSection;
+import hw6.site.elements.MetalsColorsForm;
+import lombok.Getter;
+
+@Url("/metals-colors.html")
+@Title("Metal and Colors")
+public class MetalsColorsPage extends WebPage {
+
+    @Getter private Header header;
```
Comment (troe88) [inline src/main/java/hw6/site/pages/MetalsColorsPage.java:16]: This is not the best idea to type annotation on the same line with field

**Review by troe88 — CHANGES_REQUESTED** [review]

File: src/main/java/hw6/site/elements/LoginForm.java (hunk 1 above)
Comment (obitkin) [reply src/main/java/hw6/site/elements/LoginForm.java]: Made one annotation per line

File: src/main/java/hw6/site/pages/MetalsColorsPage.java (hunk 2 above)
Comment (obitkin) [reply src/main/java/hw6/site/pages/MetalsColorsPage.java]: All annotations is over the fields for now

**Review by troe88 — APPROVED** [review]

**Pull request merged**